plugin/trino-iceberg/pom.xml

```diff
@@ -360,10 +478,28 @@
             <artifactId>testng</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.xerial</groupId>
+            <artifactId>sqlite-jdbc</artifactId>
+            <version>3.36.0.3</version>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
 
     <build>
         <plugins>
+            <plugin>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <configuration>
+                    <ignoredDependencies>
+                        <ignoredDependency>org.apache.iceberg:iceberg-aws</ignoredDependency>
+                        <ignoredDependency>io.airlift:http-client</ignoredDependency>
```

Comment:
Why is it needed? Why is it safe to do?

Reply:
I was able remove these after reorganizing the dependencies.  Should be good without now.